nautobot/ui/src/components/Apps.js

```diff
@@ -63,6 +53,20 @@ function get_components() {
                     }
                 );
             }
+            // Add Plugin Provided View Components
+            if (value?.default?.routes) {
+                // eslint-disable-next-line
+                value?.default?.routes.forEach(({ groups }) => {
+                    groups.forEach(({ items }) => {
+                        items.forEach(({ namespace, component }) => {
+                            const route = `${slugify(app_name)}:${namespace}`;
```

Comment:
I'm not a fan of using `slugify(app_name)` here - I'd prefer that the route be explicitly specified in some other fashion rather than assuming that a given app's routes follow a particular arbitrary pattern.